### ISSUE TITLE
`browse_website` does not gracefully handle browsing errors

### ISSUE THREAD

**Issue by Pwuts** [Significant-Gravitas/Auto-GPT#3401]
It seems that even simple things can throw really ugly errors:

URL = `barrel-roll.com`
```
NEXT ACTION:  COMMAND = browse_website ARGUMENTS = {'url': 'barrel-roll.com', 'question': 'How to execute a barrel roll'}
Enter 'y' to authorise command, 'y -N' to run N continuous commands, 'n' to exit program, or enter feedback for ...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
[WDM] - Downloading: 100%|███████████████████████████████████████████████████████████████████████████████████████| 6.75M/6.75M [00:01<00:00, 6.44MB/s]
SYSTEM:  Command browse_website returned: Error: Message: invalid argument (Session info: headless chrome=112.0.5615.138)
Stacktrace:
#0 0x56413e60afe3 <unknown> #1 0x56413e349bc1 <unknown> #2 0x56413e334446 <unknown>
#3 0x56413e3327f3 <unknown> #4 0x56413e332c3d <unknown> #5 0x56413e34bb16 <unknown>
#6 0x56413e3c08c5 <unknown> #7 0x56413e3a78c2 <unknown> #8 0x56413e3c0232 <unknown>
#9 0x56413e3a7693 <unknown> #10 0x56413e37a03a <unknown> #11 0x56413e37b17e <unknown>
#12 0x56413e5ccdbd <unknown> #13 0x56413e5d0c6c <unknown> #14 0x56413e5da4b0 <unknown>
#15 0x56413e5d1d63 <unknown> #16 0x56413e5a4c35 <unknown> #17 0x56413e5f5138 <unknown>
#18 0x56413e5f52c7 <unknown> #19 0x56413e603093 <unknown> #20 0x7fa787a00ea7 start_thread
```

Connection error:
```
NEXT ACTION:  COMMAND = browse_website ARGUMENTS = {'url': 'https://barrel-roll.com', 'question': "Can this website help me achieve the user's goal?"}
Enter 'y' to authorise command, 'y -N' to run N continuous commands, 'n' to exit program, or enter feedback for ...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
[WDM] - Downloading: 100%|███████████████████████████████████████████████████████████████████████████████████████| 6.75M/6.75M [00:01<00:00, 6.20MB/s]
SYSTEM:  Command browse_website returned: Error: Message: unknown error: net::ERR_CONNECTION_CLOSED (Session info: headless chrome=112.0.5615.138)
Stacktrace:
#0 0x561c26ed8fe3 <unknown> #1 0x561c26c17d36 <unknown> #2 0x561c26c0fcd8 <unknown>
#3 0x561c26c02862 <unknown> #4 0x561c26c03f72 <unknown> #5 0x561c26c02c2a <unknown>
#6 0x561c26c01cf8 <unknown> #7 0x561c26c01b30 <unknown> #8 0x561c26c00608 <unknown>
#9 0x561c26c00c3d <unknown> #10 0x561c26c19b16 <unknown> #11 0x561c26c8e8c5 <unknown>
#12 0x561c26c758c2 <unknown> #13 0x561c26c8e232 <unknown> #14 0x561c26c75693 <unknown>
#15 0x561c26c4803a <unknown> #16 0x561c26c4917e <unknown> #17 0x561c26e9adbd <unknown>
#18 0x561c26e9ec6c <unknown> #19 0x561c26ea84b0 <unknown> #20 0x561c26e9fd63 <unknown>
#21 0x561c26e72c35 <unknown> #22 0x561c26ec3138 <unknown> #23 0x561c26ec32c7 <unknown>
#24 0x561c26ed1093 <unknown> #25 0x7fcdddaeaea7 start_thread
```

_Originally posted by @Pwuts in https://github.com/Significant-Gravitas/Auto-GPT/issues/2543#issuecomment-1519194041_

**Comment by tuwid** [comment]
I've seen this behavior mostly with Chrome. If you change it to firefox one it doesn't happen but it's bothersome to have the browser open up

**Comment by Dr33dM3** [comment]
@Pwuts, this issue has become obsolete since @collijk addressed it in their pull request.

**Comment by Pwuts** [comment]
Fixed by #3494
